modules/installation-restricted-network-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc
+
```

Comment:
Outbound routing can be used in other use cases as well. But specifically for restricted network scenario, only user defined outbound routing should be used.

Reply:
> only user defined outbound routing should be used.

I think this is what has me confused. As currently written [1], User-defined outbound routing reads as an optional step. I am interpreting your comment that as part of the installation user-defined routing must be used. Do I have that correct? That is users _must_ update `platform.azure.outboundType` from `loadBalancer` to `UserDefinedRouting` prior to deploying the cluster.

Reply:
@xenolinux Quick question. I just happened to notice that you created an entirely new module for this topic, which is based on `modules/installation-azure-user-defined-routing`.  In fact the only difference between your new module and the latter are lines 27-30. Is there a reason you chose to create a new module instead of reusing the existing content?

Reply:
> Update: Or in lieu of user defined outbound routing, I can use network security groups to control access to the internet? In which case a user could leave platform.azure.outboundType set to loadBalancer Do I have this right?

Yes that is correct. I misspoke regarding user defined routing being the only option for restricted network. It should be used when the Azure Firewall is being used to restrict access to the internet. When using network security group for restricting access to the internet then the outbound type can be loadBalancer.

Reply:
Ack'ed. Removed the newly created module and added conditionals to the existing one - `modules/installation-azure-user-defined-routing`.